python/lsst/afw/cameraGeom/camera.py

```diff
@@ -28,24 +28,35 @@
 class Camera(DetectorCollection):
     """!A collection of Detectors that also supports coordinate transformation
     """
-    def __init__(self, name, detectorList, transformMap):
+    def __init__(self, name, detectorList, transformMap, pupilFactoryClass):
```

Comment:
>Can this be defaulted to the ABC?

Sure.  I'll go ahead and eliminate the PRs for the obs packages (and I think pipe_tasks too) that are no longer required.